README.md

```diff
@@ -88,8 +88,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
-        ruby: ['2.7', '3.0', '3.1', head, jruby, jruby-head, truffleruby, truffleruby-head]
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.2'
```

Comment:
This change is incorrect, can you undo it?

Reply:
done